docpipeline/generate.py

```diff
@@ -108,26 +121,41 @@ def add_prettyprint(output_path):
             file_handle.write(html)
 
 
+def is_blob(blob):
```

Comment:
I'm happy to move forward with this PR as-is. But, I wonder if there is a refactor we could do to avoid needing `is_blob`. Right now, `setup_docfx` does [something depending on blob/not] then common stuff. Then, `process_blob` does common stuff then [something depending on blob/not].

So, it seems like we could split those blob/not branches into separate functions.

Reply:
I've refactored it to local/bucket bits so that `is_blob` is no longer needed!